authentik/policies/models.py

```diff
@@ -48,6 +48,13 @@ def __str__(self) -> str:
         return f"PolicyBindingModel {self.pbm_uuid}"
 
 
+class BoundPolicyQuerySet(models.QuerySet):
+    """QuerySet for filtering enabled bindings for a Policy type"""
+
+    def for_policy(self, policy: "Policy"):
+        return self.filter(policy__in=policy._default_manager.all()).filter(enabled=True)
```

Comment:
Just to make sure I understand, why are we referring to the default manager here?

Reply:
I chose `_default_manager` because I wanted to preserve the behavior of the `InheritanceManager()` managers on the different `Policy` subclasses. Is that unnecessary here?